Bintray publish task fails when releasing new artifact versions

Although the submodule artifacts are successfuly uploaded to the Bintray, the Travis job fails due to error on running `bintrayUpload` task on root project.

```
> Task :bintrayUpload
Repository name, package name or version name are null for project: root project 'memcached-spring-boot'
> Task :bintrayPublish FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':bintrayPublish'.
> java.lang.NullPointerException (no error message)
```
This does not seem to occur when releasing **SNAPSHOT** version with `artifactoryPublish` task.
For this reason, we should exclude the `bintrayPublish` task being run on root project in the Travis build script.

For more details about the build error in Travis check the [logs](https://travis-ci.org/github/sixhours-team/memcached-spring-boot/jobs/667994079)

--- a/travisBuild.sh
+++ b/travisBuild.sh
@@ -13,7 +13,7 @@ elif [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_TAG" != "" ]; then
   echo -e 'Build Branch for Release => Branch ['$TRAVIS_BRANCH']  Tag ['$TRAVIS_TAG']'
   case "$TRAVIS_TAG" in
   v[[:digit:]]*\.[[:digit:]]*\.[[:digit:]]*)
-    ./gradlew build bintrayUpload -PuseLastTag=true
+    ./gradlew build bintrayUpload -x :bintrayUpload -PuseLastTag=true
     ;;
   *)
     echo -e 'WARN: Invalid Tag ['$TRAVIS_TAG']'

